backbone.queryparams.js

```diff
@@ -360,7 +360,11 @@ function iterateQueryString(queryString, callback) {
 }
 
 function containsInvalidKey(keys) {
```

Comment:
The argument `keys` should be renamed to `key`, right?
Even better, maybe this method should be renamed to `isInvalid(key)`. WDYT?